### PR TITLE
feature: support zstd compressor

### PR DESCRIPTION
**Pull request by caohdgege**

### CODE REVIEW

File: bom/pom.xml [hunk 1]
```diff
@@ -372,6 +372,31 @@
                 <artifactId>seata-sqlparser-druid</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>com.github.luben</groupId>
+                <artifactId>zstd-jni</artifactId>
```
Comment (a364176773) [inline bom/pom.xml:377]: bom挪到dependencies

**Review by slievrly — COMMENTED** [review]
add artifactSet in `maven-shade-plugin`

**Review by a364176773 — APPROVED** [review]
LGTM

**Review by slievrly — APPROVED** [review]
LGTM

**Pull request merged**